.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
**TODO FOR LOST POINTS**

This is a top level comment that you need at least one placeholder image somewhere just to see how images will scale with your app. I suggest using this tool for placeholder images:
https://placehold.it/